### PR TITLE
Update README with Ubuntu dependencies

### PR DESCRIPTION
**Pull request by amirouche**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,11 @@ On a Mac with [Homebrew](http://brew.sh/), run
 brew install cmake ninja boost libpng glfw3 libepoxy guile
 ```
 
+On Ubuntu:
+```
+sudo apt install cmake ninja-build libboost-all-dev libpng16-dev libglfw3-dev libepoxy-dev guile-2.0
```
Comment (daveloyall) [inline README.md]: should `apt` be `apt-get`?  It *should* be `apt-get` on Debian.  Not sure what Ubuntu is like these days!

Comment (sethalves) [inline README.md]: I think I also needed guile-2.0-dev guile-2.0-libs

Comment (amirouche) [inline README.md]: @mkeeter `apt` is available both on Debian Jessie (stable) and Ubuntu

@ervanalb Thx!

**Comment by amirouche** [comment]
I've redone the commit. `apt` is available both on Debian Jessie (stable) and Ubuntu.

**Comment by mkeeter** [comment]
Thanks!

**Pull request merged**